benchmark/clblast/cli_device_selector.hpp

```diff
@@ -38,65 +38,34 @@
 #include <utility>
 #include <vector>
 
+#include "benchmark_cli_args.hpp"
+
 class cli_device_selector {
   std::string program_name;
 
-  void usage() {
-    std::cout << "Usage: " << program_name << " [google-benchmark args]"
-              << std::endl;
-    std::cout << " --device  DEVICE" << std::endl
-              << "    Select a device (best effort) for running the benchmark."
-              << std::endl;
-    std::cout << "    e.g. intel:cpu, amd:gpu etc" << std::endl;
-  }
-
  public:
-  std::string device_vendor;
+  std::string vendor_name;
```

Comment:
Is there any reason why this was changed? I somewhat prefer the whole device_<...> naming scheme, but if there's a reason to change it, then do so.

Reply:
To use the same names in both CLBLAST and SYCL-BLAS device selectors. I can revert that if you prefer though